starport/services/network/launch.go

```diff
@@ -0,0 +1,59 @@
+package network
+
+import (
+	"context"
+	"fmt"
+
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+	"github.com/tendermint/starport/starport/pkg/cosmosutil"
+	"github.com/tendermint/starport/starport/pkg/events"
+	"github.com/tendermint/starport/starport/services/network/networkchain"
+)
+
+// LaunchParams fetches the chain launch module params from SPN
+func (n Network) LaunchParams(ctx context.Context) (launchtypes.Params, error) {
+	res, err := launchtypes.NewQueryClient(n.cosmos.Context).Params(ctx, &launchtypes.QueryParamsRequest{})
+	if err != nil {
+		return launchtypes.Params{}, err
+	}
+	return res.GetParams(), nil
+}
+
+// Launch launch a chain as a coordinator
+func (n Network) Launch(ctx context.Context, launchID, remainingTime uint64) error {
```

Comment:
I believe we decided to call the command `chain launch` to be more simple since this will be the command that will make the chain launched

Though this specific method action is still to just trigger the launch with `MsgTriggerLaunch` so I would call it `TriggerLaunch`